app/controllers/api/v1/users_controller.rb

```diff
@@ -11,16 +11,12 @@ def create
       @user = existing_user ## Lead that hasnt created their own password
     end
     @user.assign_attributes(user_params)
-    @user.employer = Organization.where(
-      title: organization_params[:organization_title]
-    ).first_or_initialize
-    @user.role = :basic
-
+    @user.employer = define_employer
+    @user.role = :basic unless @user.role.eql?(:admin)
```

Comment:
Tried to make this a default param in the model but realized we use it to confirm switches from lead to basic. I slipped this in bc it actually messes up inviting admins as it was converting them to basic, too.

Reply:
Do you think we still need the lead/basic distinction? I had set it up originally so we could identify users who needed to set a password, but we are using the `joined?` method instead for that.  Not sure if it is still helpful to have these separate roles?

If we leave it, then line 15 should be:

```
@user.role = :basic unless @user.admin?
```

Reply:
Let's leave it as-is for now. We can rip it up later.